tests/hikari/impl/test_shard.py

```diff
@@ -1210,7 +1210,7 @@ async def test_when_opcode_is_HEARTBEAT_ACK(self, client, exit_error):
             "d": "some data",
         }
         client._receive_json = mock.AsyncMock(side_effect=[payload, exit_error])
-        client._last_heartbeat_sent = 5
+        client._last_sent = 5
```

Comment:
Refactoring mistake, but pretty much all of this file is now useless again, so meh. 

Will have to rewrite all of this as part of this PR, when I get time.